src/time.jl

```diff
@@ -23,13 +23,9 @@ function every(dt)
     n
 end
 
-function weakrefdo(ref, yes, no=()->nothing)
```

Comment:
Or compute the correct frame from a function such as `update(current_frame, dt) -> next_frame`. The animation would then just be: `foldp(update, init_frame, fps(60))`, of course you are also free to use `time()` and `map`  with `fps`.